src/api/utils/handleResponseData.spec.ts

```diff
@@ -1,6 +1,6 @@
 import APIError from 'utils/apiError';
 import handleResponseData from './handleResponseData';
-import mockFetchResponse from 'test-utils/mocks/mockFetchResponse';
+import mockFetchResponse from 'utils/testing/mocks/mockFetchResponse';
```

Comment:
I like this better, but mostly because it's what I'm used to doing. 😊

Reply:
Do you mean you prefer test-utils still in its own folder? :)

Reply:
No I like it the way you've done it! 😊